src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -160,7 +161,18 @@ def _populate_core_stats(self, area):
         if hasattr(area.current_market, 'trades'):
             for trade in area.current_market.trades:
                 core_stats_dict['trades'].append(trade.serializable_dict())
-        if getattr(area, 'strategy', None) is None and area.current_market is not None:
+        if area.strategy is None and area.current_market is not None:
+            area_stats = {
```

Comment:
Here too, you dont need to assign a separate variable if only needed once.

Reply:
Okay, will do so, if you prefer that way.